src/Core/Framework/App/Manifest/ModuleLoader.php

```diff
@@ -57,10 +64,33 @@ public function loadModules(Context $context): array
         /** @var AppCollection $apps */
         $apps = $this->appRepository->search($criteria, $context)->getEntities();
 
-        return $this->formatPayload($apps);
+        if ($context->getLanguageIdChain() !== []) {
```

Comment:
Thats the wrong way. You use the content language as user language. 
Try to pick it from the UserSource -> Admin User -> Choosen locale

Reply:
Aaaah snap, you are right. But that is not a parameter yet to the request.

Reply:
Call this method
https://github.com/shopware/platform/blob/d6c853ca3b6b167d61d70e6329f4fcface5c5245/src/Core/Framework/Store/Services/StoreService.php#L97

Reply:
Great hint